lib/normalize.js

```diff
@@ -55,14 +53,14 @@ const nomralizeDateFormat = (dp) => {
 module.exports.nomralizeDateFormat = nomralizeDateFormat
 
 const normalizeType = (dp) => {
-  for (let key1 in dp.resources){
-    let value1 = dp.resources[key1]
-    for (let key2 in value1.schema.fields){
-      let value2 = value1.schema.fields[key2]
+  for (let resourceName in dp.resources){
+    let resource = dp.resources[resourceName]
+    for (let fieldName in resource.schema.fields){
+      let field = resource.schema.fields[fieldName]
       let unsupported_number_types = ['decimal', 'double', 'float']
       for (let i = 0; i < unsupported_number_types.length; i++) {
-        if (unsupported_number_types[i] === value2.type) {
-          value2.type = 'number'
+        if (unsupported_number_types[i] === field.type) {
+          field.type = 'number'
```

Comment:
I think it would be simpler to write
```
if (unsupported_number_types.indexOf(err.response.status) > -1) {
  field.type = 'number'
}
```
instead of all line 60 - 65